index.php

```diff
@@ -24,29 +25,42 @@
 // sets the searchq variable to whatever is in the input field or in the url if it isn't empty, if it is then leave it blank
 if (isset($_GET['q']) ? $searchq = $_GET['q'] : $searchq = '');
 
-session_start();
-
-
+if (isset($_SESSION[SESSION_KEY_USER_ID])) {
+    echo $_SESSION[SESSION_KEY_USER_ID];
+}
 $routes = [
-    '/' => 'controllers/index.php',
+    '/dashboard' => 'controllers/index.php',
```

Comment:
waarom maak je nog een dashboard aan in de routes?

Reply:
had ook / kunnen zijn. maar vond ik mooi :)

Reply:
alright